spot_wrapper/tests/test_wrapper.py

```diff
@@ -120,3 +122,144 @@ def test_spot_wrapper_commands(simple_spot: SpotFixture, simple_spot_wrapper: Sp
     assert not ok
     ok, _ = simple_spot_wrapper.sit()
     assert not ok
+
+
+def test_mobility_command_feedback(simple_spot: SpotFixture, simple_spot_wrapper: SpotWrapper) -> None:
+    non_processing_feedback_statuses: List[RobotCommandFeedbackStatus] = [
+        RobotCommandFeedbackStatus.STATUS_UNKNOWN,
+        RobotCommandFeedbackStatus.STATUS_COMMAND_OVERRIDDEN,
+        RobotCommandFeedbackStatus.STATUS_COMMAND_TIMED_OUT,
+        RobotCommandFeedbackStatus.STATUS_ROBOT_FROZEN,
+        RobotCommandFeedbackStatus.STATUS_INCOMPATIBLE_HARDWARE,
+    ]
+
+    # bosdyn.api.RobotCommandService/RobotCommand implementation is mocked
+    # (as spot_wrapper.testing.mocks.MockSpot is automatically specified),
+    # so here we provide the same response for every future command.
+    response = RobotCommandResponse()
+    response.status = RobotCommandResponse.Status.STATUS_OK
+    simple_spot.api.RobotCommand.future.returns(response).forever()
+
+    def update_with_feedback_response(r: RobotCommandFeedbackResponse) -> None:
+        simple_spot.api.RobotCommandFeedback.future.returns(r).repeatedly(1)
```

Comment:
@Imaniac230 fyi: 
```suggestion
        simple_spot.api.RobotCommandFeedback.future.returns(r)
```
is equivalent to what you have.

Reply:
That never occurred to me, thanks.